concent_api/gatekeeper/utils.py

```diff
@@ -1,8 +1,8 @@
 from django.http import JsonResponse
 
 
-def gatekeeper_access_denied_response(message):
-    data = {'message': message}
+def gatekeeper_access_denied_response(message, path = None, subtask_id = None, client_key = None):
+    data = {'message': message, 'path to file': path, 'subtask id': subtask_id, "client's key": client_key}
```

Comment:
1) Don't use spaces or apostrophes in key names in JSON responses.
2) If you have more than one key in a dict it's better to wrap it into multiple lines. Otherwise it's hard to read.

Reply:
Also, you are mixing "" with ''. Please choose one and use it consistently.

Reply:
I'm all for consistency but please keep in mind one exception: always use the double quotes when there are single quotes inside. So for example `"That's an error. Let's fix this 'issue'."` is better than `'That\'s an error. Let\'s fix this \'issue\'.` even if all the other strings around use single quotes. In general I think that it's a good idea to always use double quotes for messages because they often contain apostrophes.

Reply:
Okey, I've changed names of all keys, so they don't have any spaces and apostrophes. They can all stay like this: '<key>'.